src/Capsule/Collection.php

```diff
@@ -206,6 +206,10 @@ private function prepareQuery(string $method, $filters, $data, array $options):
 
     private function translateReadPreference(ReadPreference $readPreference): string
     {
+        if (version_compare(phpversion('mongodb'), '1.20.0', '>=')) {
+            return $readPreference->getModeString();
+        }
```

Comment:
Couldn't you check for the existence of the method? It should be faster and clearer.
```suggestion
        if (method_exists(ReadPreference::class, 'getModeString')) {
            return $readPreference->getModeString();
        }
```